wallets-testing/test/test.service.ts

```diff
@@ -1,19 +1,29 @@
 import { configService } from '../config';
-import { EthereumNodeService } from '@lidofinance/wallets-testing-nodes';
 import { ETHEREUM_WIDGET_CONFIG } from '@lidofinance/wallets-testing-widgets';
-import { ExtensionService } from '@lidofinance/wallets-testing-extensions';
-import { BrowserContextService, BrowserService } from '../browser';
+import { BrowserService } from '@lidofinance/browser-service';
+import { METAMASK_COMMON_CONFIG } from '@lidofinance/wallets-testing-wallets';
 
 export async function initBrowserService() {
-  const ethereumNodeService = new EthereumNodeService({
-    chainId: ETHEREUM_WIDGET_CONFIG.chainId,
-    rpcUrl: configService.get('RPC_URL'),
-    defaultBalance: 1000,
+  METAMASK_COMMON_CONFIG.LATEST_STABLE_DOWNLOAD_LINK = undefined;
+  return new BrowserService({
+    networkConfig: {
+      chainId: ETHEREUM_WIDGET_CONFIG.chainId,
```

Comment:
I think better to use `wallets.constants.NETWORKS_CONFIG.Mainnet.ETHEREUM - chaind/chainName/tokenSymbol` since it's more usable. Btw `ETHEREUM_WIDGET_CONFIG` have to be  refactored i think. Not sure here

Reply:
where is wallets.constants.NETWORKS_CONFIG.Mainnet.ETHEREUM?

Reply:
[here](https://github.com/lidofinance/wallets-testing-modules/blob/ec164f8449727ce167c7a2da0f773d279cb75c5f/packages/wallets/src/wallets.constants.ts#L61) 

> where is wallets.constants.NETWORKS_CONFIG.Mainnet.ETHEREUM?